src/Command/ListProjectsCommand.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Pim\Bundle\TextmasterBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Textmaster\Model\Project;
+
+/**
+ * Retrieve all TextMaster projects
+ *
+ * @author    Jean-Marie Leroux <jean-marie.leroux@akeneo.com>
+ * @copyright 2016 TextMaster.com (https://textmaster.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+class ListProjectsCommand extends ContainerAwareCommand
+{
+    /** @var OutputInterface */
+    private $output;
```

Comment:
Oh yes. 
- It's a utility command, not meant to be overriden.
- even if it's overriden, i don't see any reason to override the output.